.azure/build-pipeline.yaml

```diff
@@ -32,7 +32,7 @@ stages:
           artifactRunVersion: ''
           artifactRunId: ''
     variables:
-      STRIMZI_TEST_CONTAINER_LOGGING_ENABLED: false
+      STRIMZI_TEST_CONTAINER_LOGGING_ENABLED: true
```

Comment:
Should this be kept?

Reply:
Okay so additional two questions:
- how much it will take from the storage?
- will you remember to disable it in 2 weeks? :D

Reply:
> how much it will take from the storage?

<img width="585" height="44" alt="image" src="https://github.com/user-attachments/assets/ab47433b-6621-4320-a26b-222fe937cc29" />

As I can see, it would add 4MBs. 
 
 > will you remember to disable it in 2 weeks? :D
 
 I will make a note, trust me 🎺 .

Reply:
He will open an issue! 😉